e2e/steps/match.ts

```diff
@@ -79,15 +68,6 @@ export const matchAndBookApplication = async ({
   const premisesName = await searchScreen.retrieveFirstAPName()
   await searchScreen.selectFirstAP()
 
-  // Then I should see the booking screen for that AP
-  const bookingScreen = await BookingScreen.initialize(page, premisesName)
```

Comment:
I take it that the confirmation page is no longer reachable. This would be a bit annoying as it means we can no longer create space-bookings through the UI.

How about adding a button on the new occupancy view page that jumps to the confirmation page, then you can add this to the e2e tests without needing to remove the confirmation page tests.

Reply:
okay great! I was a bit surprised we were blocking this journey entirely. I will add in the button to navigate to the confirmation page as suggest - thanks both :)